test/acceptance/clean-architecture.js

```diff
@@ -0,0 +1,133 @@
+var app = require("../../examples/clean-architecture");
+var request = require("supertest");
+
+describe("clean-architecture-crud", function () {
+  describe("GET /", function () {
+    it("should return empty array", function (done) {
+      request(app).get("/notes").expect(200, [], done);
+    });
+
+    it("list after creation", function (done) {
+      request(app)
+        .post("/notes")
+        .set("Content-Type", "application/json")
+        .send('{"title": "Text"}')
+        .expect(201, function (err, _res) {
+          if (err) return done(err);
+          request(app)
+            .get("/notes")
+            .expect(200, function (err, res) {
+              if (err) return done(err);
+              if (res.body.length === 1 && res.body[0].title === "Text") {
+                done();
+              }
```

Comment:
without an `else` here, a failure of the check leaves the test suite hanging. if you are testing the body contents, just use supertest to do it rather than manually checking res.body which would simplify it.